examples/api/lib/material/menu_bar/menu_anchor.0.dart

```diff
@@ -0,0 +1,210 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Flutter code sample for [MenuAnchor].
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() => runApp(const MenuApp());
+
+/// An enhanced enum to define the available menus and their shortcuts.
+///
+/// Using an enum for menu definition is not required, but this illustrates how
+/// they could be used for simple menu systems.
+enum MenuEntry {
+  about('About'),
+  showMessage('Show Message', SingleActivator(LogicalKeyboardKey.keyS, control: true)),
+  hideMessage('Hide Message'),
```

Comment:
is hideMessage actually used? It seems like `showMessage` is used to toggle and maybe should be renamed to that? And then `hideMessage` can be removed?

Reply:
Actually, it is now.